app/src/main/res/layout/notification_expanded.xml

```diff
@@ -46,7 +46,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/hori_small"
-            android:text="List Name"
+            android:text="@string/listName"
```

Comment:
Here too, 'List Name' is only a placeholder and will be replaced by the list's actual name at inflation/run time. So I suggest the use of 'placeholder' keyword.

Reply:
Requested changes for adding "placeholder" in string name has been done